rpc/Cargo.toml

```diff
@@ -12,6 +12,9 @@ sp-blockchain = { git = "https://github.com/paritytech/substrate", branch = "mas
 sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "master"  }
 sp-api = { git = "https://github.com/paritytech/substrate", branch = "master"  }
 sc-rpc = { git = "https://github.com/paritytech/substrate", branch = "master" }
+sc-finality-grandpa = { git = "https://github.com/paritytech/substrate", branch = "master" }
+# FIXME: must be updated before merging
+sc-finality-grandpa-rpc = { git = "https://github.com/octol/substrate", branch = "jon/issue-4921-expose-grandpa-round-state-try-shared-arc" }
```

Comment:
Update for merge
```suggestion
sc-finality-grandpa-rpc = { git = "https://github.com/paritytech/substrate", branch = "master" }
```

Reply:
I made this temporary change purposefully. The current approach of overriding with `.cargo/config` doesn't work when a crate doesn't exist (i.e. sc-finality-grandpa-rpc currently does not exist on master). Without this change the substrate companion build task will fail to build.

Reply:
yeah, just learned that the hard way. but we had CI pass, so if the review passes, we can merge non-the-less.